recipes/lodepng/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.23)
+
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+
+project(lodepng
+        VERSION "$ENV{PKG_VERSION}"
+        LANGUAGES CXX)
+
+# To build shared libraries in Windows, we set CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS to TRUE.
+# See https://cmake.org/cmake/help/v3.4/variable/CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS.html
+# See https://blog.kitware.com/create-dlls-on-windows-without-declspec-using-new-cmake-export-all-feature/
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+add_library(lodepng lodepng.cpp)
+
+target_include_directories(lodepng PUBLIC
+  $<INSTALL_INTERFACE:include>
+)
+
+set(LODEPNG_CMAKE_INSTALL_DIR lib/cmake/lodepng)
+
+file(WRITE "${CMAKE_BINARY_DIR}/lodepng-config.cmake" 
+           "include(\"\${CMAKE_CURRENT_LIST_DIR}/lodepng-targets.cmake\")")
+           
+write_basic_package_version_file("${CMAKE_BINARY_DIR}/lodepng-config-version.cmake" 
+                                 COMPATIBILITY AnyNewerVersion)
```

Comment:
Any newer version!? That's a great compatibility guarantee. They will never remove things from the API and never break the ABI? 😮

Reply:
Obviously not (and this is anyhow not related to ABI, just API). It is just that in my experience the `find_package` behaviour provided by `AnyNewerVersion` is what users expect. Anyhow, I do not feel strongly about that, we can switch to `ExactVersion`.